backend/presentation/models.py

```diff
@@ -0,0 +1,33 @@
+import uuid
+
+from django.contrib.auth import get_user_model
+from django.db import models
+
+User = get_user_model()
+
+
+class Tag(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.name
+
+
+class Presentation(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
```

Comment:
If `id` is specified here, should we keep `default_auto_field` in this app's config?